include/manifold/common.h

```diff
@@ -13,59 +13,71 @@
 // limitations under the License.
 
 #pragma once
-#define GLM_ENABLE_EXPERIMENTAL  // needed for glm/gtx/compatibility.hpp
-#define GLM_FORCE_EXPLICIT_CTOR
-#include <glm/ext/matrix_transform.hpp>
-#include <glm/glm.hpp>
-#include <glm/gtc/constants.hpp>
-#include <glm/gtx/compatibility.hpp>
-#include <glm/gtx/rotate_vector.hpp>
 #include <limits>
 #include <stdexcept>
 #include <vector>
 
+#include "linalg.h"
+
 namespace manifold {
 /** @defgroup Math data structure definitions
- *  @brief Abstract away from glm.
+ *  @brief Abstract away from linalg.
  *  In the future the underlying data type can change.
  *  @{
  */
-using vec2 = glm::dvec2;
-using vec3 = glm::dvec3;
-using vec4 = glm::dvec4;
-using mat2 = glm::dmat2;
-using mat2x3 = glm::dmat2x3;
-using mat2x4 = glm::dmat2x4;
-using mat3x2 = glm::dmat3x2;
-using mat3 = glm::dmat3;
-using mat3x4 = glm::dmat3x4;
-using mat4x3 = glm::dmat4x3;
-using mat4 = glm::dmat4;
-using ivec2 = glm::vec<2, int>;
-using ivec3 = glm::vec<3, int>;
-using ivec4 = glm::vec<4, int>;
-using quat = glm::dquat;
+namespace la = linalg;
```

Comment:
Not sure if we want this namespace alias in our public header. Maybe define this in our internal header?

Reply:
It seems that this is fine, we are having `la` alias inside our manifold namespace, so this will not cause conflict elsewhere.